src/main/webapp/js/instructorStudentRecords.es6

```diff
@@ -122,6 +126,8 @@ function checkComment(form) {
     }
 }
 
+/* eslint-enable consistent-return */
```

Comment:
Warning: Replace `return false;` in event handlers with `e.preventDefault(); e.stopPropagation();`.

I just got burned by this.